sktime/transformations/base.py

```diff
@@ -1234,7 +1236,7 @@ def _convert_output(self, X, metadata, inverse=False):
             Xt = convert_to(
                 Xt,
                 to_type=X_output_mtype,
-                as_scitype=X_input_scitype,
```

Comment:
@fkiraly, I would appreciate your input here.

The merger is transforming a Panel into a Series. I believe that this transformation is not supported by _convert_output. Thus, I changed it. However, I am very unsure if this is done correctly and would like to have your advise here.

Furthermore, I was confuses that the output is transformed to the `X_input_scitype` and not to the `output_scitype`.